modules/lang-mustache/src/internalClusterTest/java/org/elasticsearch/script/mustache/SearchTemplateIT.java

```diff
@@ -189,6 +191,64 @@ public void testIndexedTemplateClient() throws Exception {
         assertNull(getResponse.getSource());
     }
 
+    public void testBadTemplate() {
```

Comment:
small nit: could you add a short description of this test for future audiences, i.e. what is wrong with the template and why it should fail? Also that we test that both the simulate/non-simulate options return the same error.